wandb/sdk/launch/utils.py

```diff
@@ -66,19 +68,13 @@ def set_project_entity_defaults(
     else:
         uri_project = UNCATEGORIZED_PROJECT
     if project is None:
-        project = uri_project or UNCATEGORIZED_PROJECT
-        wandb.termlog(
-            "Target project for this run not specified, defaulting to project {}".format(
-                project
-            )
-        )
+        project = api.settings("project") or uri_project or UNCATEGORIZED_PROJECT
     if entity is None:
         entity = api.default_entity
-        wandb.termlog(
-            "Target entity for this run not specified, defaulting to current logged-in user {}".format(
-                entity
-            )
-        )
+    prefix = ""
+    if platform.system() != "Windows" and sys.stdout.encoding == "UTF-8":
+        prefix = "🚀 "
+    wandb.termlog("{}Launching run into {}/{}".format(prefix, entity, project))
```

Comment:
I just consolidated these messages into one telling the users where the launched run will end up.